packages/components/src/components/Header/Header.js

```diff
@@ -11,26 +11,65 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import React from 'react';
+import React, { Component } from 'react';
 
 import { Link } from 'react-router-dom';
-import { Header as CarbonHeader } from 'carbon-components-react';
-
+import { Button, Header as CarbonHeader } from 'carbon-components-react';
+import LogoutIcon from '@carbon/icons-react/lib/logout/20';
 import './Header.scss';
+import { shouldDisplayLogout } from '../../../../../src/api';
```

Comment:
code in packages/* should not consume code from the dashboard core
instead, the API function required should be passed in as a prop when the component is used

this ensures we can re-use the components in other contexts outside the dashboard

Reply:
Thanks, lemme give this a go